src/tls13.c

```diff
@@ -8218,6 +8218,85 @@ int wolfSSL_request_certificate(WOLFSSL* ssl)
 }
 #endif /* !NO_CERTS && WOLFSSL_POST_HANDSHAKE_AUTH */
 
+
+#if defined(OPENSSL_EXTRA) && defined(HAVE_ECC)
```

Comment:
Typically we put the compatibility functions in ssl.c. Perhaps move to below `wolfSSL_EC_curve_nid2nist`?